AAEmu.UnitTests/Game/Core/Managers/AccessLevelManagerTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+
+using AAEmu.Game.Core.Managers;
+using AAEmu.Game.Models;
+
+using Xunit;
+
+namespace AAEmu.UnitTests.Game.Core.Managers
+{
+    public class AccessLevelManagerTests
+    {
+        private readonly AccessLevelManager _manager;
+
+        public AccessLevelManagerTests()
+        {
+            _manager = new AccessLevelManager();
+            ResetAppConfiguration();
+        }
+
+        private void ResetAppConfiguration()
+        {
+            var config = AppConfiguration.Instance;
+            var accessLevelProperty = config.GetType().GetProperty("AccessLevel");
+            accessLevelProperty.SetValue(config, new Dictionary<string, int>());
```

Comment:
I think the reflection here isn't needed - the AccessLevel property has a setter, so could be set directly:
```suggestion
            AppConfiguration.Instance.AccessLevel = new Dictionary<string, int>();
```

But the dictionary is also mutable, so you could just clear it:
```suggestion
            AppConfiguration.Instance.AccessLevel?.Clear();
```